README.md

```diff
@@ -73,11 +73,19 @@ Download `xtensor` as test package:
 ./download-test-package.sh
 ```
 
-Run test upload using quetz-client. (For testing purposes, an API key is created for the test user "alice" at server launch and is printed to the terminal, so use that for this example):
+To upload the package, install the [quetz-client](https://github.com/mamba-org/quetz-client):
+
+```bash
+mamba install quetz-client
+```
+
+To run the upload, you need to set environment variables for the quetz API key (which authenticates you) and the quetz server URL. As we passed the `--dev` flag, a testing API key can be found in quetz's output which you can use for this example.
```

Comment:
What `--dev` flag are you referring to?

Reply:
quetz has a `--dev` flag that populates the db with users, packages etc.